### DISCUSSION TITLE
Depoying dcm4chee-arc: problem with configuration for deviceName

### DISCUSSION THREAD

**Discussion by rafzio4** [dcm4che/dcm4chee-arc-light#3645] [General]
Hallo,

  while deploying I get following error: 
 Missing Configuration for Device 'dcm4chee-arc' - you may change the Device name by System Property 'dcm4chee-arc.DeviceName'
System properties in dcm4chee-arc.xml:
<system-properties>
        <property name="dcm4chee-arc.deviceName" value="dcm4che.org"/>
    </system-properties>
whatever value is, I get the same error with 'dcm4chee-arc'. There aren't any entries in my xml with this name.
While setup of OpenDJ there is a field for importing file, but the instructions at https://github.com/dcm4che/dcm4chee-arc-light/wiki/Installation doesn't mention witch file should I use.

please help, 

with regards

Rafal Z

**Comment by vrindanayak** [comment]
Are you sure the name of your archive device in LDAP is `dcm4che.org` - it looks incorrect as system property definition in your Wildfly. Please show your LDAP configuration and Wildfly configuration file.

**Comment by arumbala72** [comment]
Hi Vrinda, I would like to change deviceName=dcm4chee-arc to new name, how do i do that?
